bin/webp.dart

```diff
@@ -233,13 +254,56 @@ void logUsage(ArgParser argParser) {
   log(argParser.usage);
 }
 
+Future<String> getPath() async {
+  if (!architectures.keys.contains(Abi.current())) {
+    final m1 = 'Architecture ${Abi.current()} not spported.';
```

Comment:
Typo: spported.

Is `m1/2` a short for message?

Reply:
Yes, in the editor I didn't want to view the whole message as a loooong line, so I've created temps. I can make it `msg1/2`.

Reply:
In the first place, supported architectures were listed here, I guess I can make it look nicer now